src/sql/run.py

```diff
@@ -126,6 +126,14 @@ def _repr_html_(self):
         if self.pretty:
             self.pretty.add_rows(self)
             result = self.pretty.get_html_string()
+            EXTRA = (
+                "%s\n<span style='font-style:italic;font-size:small'>"
+                "<code>ResultSet</code> : "
+                "to convert to pandas, call <code>.DataFrame()</code> "
+                "or to polars, call <code>.PolarsDataFrame()</code></span><br>"
```

Comment:
looks better, let's make it a bit smaller

also, let's add some links. click on DataFrame() should take to https://jupysql.ploomber.io/en/latest/integrations/pandas.html

and clicking on PolarsDataFrame should take to https://jupysql.ploomber.io/en/latest/integrations/pandas.html